src/commands/LoadController.php

```diff
@@ -184,12 +184,35 @@ protected function process($data)
             }
         }
 
-        foreach (ArrayHelper::getValue($data, 'assign', []) as $userid => $permissionOrRoles) {
+        $assignData = ArrayHelper::getValue($data, 'assign', []);
+        $findBy = ArrayHelper::remove($assignData, 'findBy');
+        foreach ($assignData as $userid => $permissionOrRoles) {
+            Yii::info(sprintf("User %s start:", $userid));
+            if(!is_null($findBy)) {
+                try {
```

Comment:
Instead of doing a try/catch in the loop in case of the function provide by `findBy` doesn't exist we can check if it exist before and stop the program with an error message before the loop like [`method_exists(Yii::$app->user->identityClass, $findBy)`](https://www.php.net/manual/en/function.method-exists.php)

Reply:
I think using `method_exists()` to check function provided by `findBy` is not neccesary because the function `call_user_func_array()` below will cover this scenario. It throws error message `call_user_func_array() expects parameter 1 to be a valid callback, class 'lbn\yii\keycloak\models\User' does not have a method 'findIdentitys'` and stop the program. Thus we can decrease 1 or 2 line(s) of code.

Reply:
Less expensive to properly check than to wait for an exception to be triggered.